src/com/palmergames/bukkit/towny/TownySettings.java

```diff
@@ -39,10 +55,7 @@
 	// private static Pattern namePattern = null;
 	private static CommentedConfiguration config, newConfig, language, newLanguage;
 
-//  Creatorfromhell's PR for replacing SortedMap town and nation levels.
-//	private static final NavigableMap<Integer, Map<TownLevel, Object>> configTownLevel = Collections.synchronizedNavigableMap(new TreeMap<Integer, Map<TownySettings.TownLevel, Object>>(Collections.reverseOrder()));
-//	private static final NavigableMap<Integer, Map<TownySettings.NationLevel, Object>> configNationLevel = Collections.synchronizedNavigableMap(new TreeMap<Integer, Map<TownySettings.NationLevel, Object>>(Collections.reverseOrder()));
-
+	// Creatorfromhell's PR
```

Comment:
Might as well nuke this line too.

Reply:
Done in next commit also just fixed something I saw when I was working on TownyNameUpdater in the enum